cdn/single-origin.yml

```diff
@@ -436,7 +499,7 @@ Resources:
                     - TLSv1.1
                     - TLSv1
             DomainName: !Ref OriginDomain
-            Id: web-origin
+            Id: primary-origin
```

Comment:
Will this tear down the origin for existing stacks?  What impact does that have on cached objects?  Hopefully they wouldn't get invalidated because they came from a gone origin.

Reply:
I can't say for sure. I believe this will create a new origin record under the distribution, because I don't see anyway to set an origin ID other than when creating it (i.e., they're immutable). I don't think that would bust any caches, though. My understanding is that object key evaluation happens first within the edge, and if there's a miss only then does CloudFront attempt to get the resource from an origin.

Reply:
I also don't know that we have any distributions based on this template that would be particularly hard-hit even if the global cache got busted. We can be safer with something like that DT bucket if/when we implement the origin groups.